view/frontend/templates/js/replacejs.phtml

```diff
@@ -434,6 +434,10 @@ ob_start();
 
         var hints = {prefill:{}};
 
+        var boltCheckoutConfig = {};
+
+        var addedToCart = false;
```

Comment:
Cam we call something a tad more descriptive like `newItemAddedToCart`. I don't think `addToCart` fully captures why flag is needed.

Reply:
done